google/resource_bigquery_dataset.go

```diff
@@ -184,6 +184,12 @@ func resourceBigQueryDataset() *schema.Resource {
 				},
 			},
 
+			"force_destroy": {
```

Comment:
I'm leaning slightly towards copying the naming the API uses, `delete_contents` instead of calling the field `force_destroy`. Or maybe `delete_contents_on_destroy`; it's a little verbose, but describes the behaviour well. What do you think?

Reply:
`delete_contents_on_destroy` sounds great to me actually. I'll make the change.